dictionary/lib/dictionary/impl.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Dictionary.Impl do
+  
+  def random_word() do
+    Agent.get(Dictionary.Store, fn store -> store end) |> Enum.random()
```

Comment:
# -3

What you're doing here is returning the whole word list from the agent to the caller, then picking one word. If instead you had

~~~ elixir
Agent.get(Dictionary.Store, &Enum.random/1)
~~~

the word would be done by the agent, and only a single word would be passed.